algorithms/C/searching/linearsearch.c

```diff
@@ -0,0 +1,67 @@
+// linear search
+#include <stdio.h>
+#define SIZE 100  //change according to question
```

Comment:
take size from the user so memory waste can be prevented and think if the size>100 then the program will terminate